src/components/StateFooter/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Box, Button, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper
+  }
+}));
+
+const StateFooter = () => {
+  const classes = useStyles();
+
+  const handleViewState = (event: any) => {};
+
+  const handleResetState = (event: any) => {};
+  
+  return (
+    <Box className={classes.root}>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="flex-end"
```

Comment:
Whether you add or remove this. It probably doesn't make a difference. So if it doesn't, you can remove it. There's no need to increase the files byte size.